packages/docz-plugin-vue/src/index.ts

```diff
@@ -0,0 +1,40 @@
+import { createPlugin } from 'docz-core'
+import { VueLoaderPlugin } from 'vue-loader'
+
+interface BabelRC {
+  presets?: any[]
+  plugins?: any[]
+  cacheDirectory?: boolean
+  babelrc?: boolean
+}
+
+type ModifyBabelRC = (babelrc: BabelRC) => BabelRC
+type ModifyBundlerConfig<C = any> = (config: C, dev: boolean) => C
+
+export const doczPluginVue = () =>
+  createPlugin({
+    modifyBabelRc: ((babelrc: BabelRC): BabelRC => {
+      if (babelrc.plugins === undefined) {
+        babelrc.plugins = []
+      }
+      babelrc.plugins.push('vuera/babel')
+      babelrc.babelrc = true
+      return babelrc
+    }) as ModifyBabelRC,
```

Comment:
I think that ts will infer this for you... you really need this `as <Something>`?